src/control/server/server.go

```diff
@@ -61,6 +61,11 @@ const maxIoServers = 2
 func Start(log *logging.LeveledLogger, cfg *Configuration) error {
 	log.Debugf("cfg: %#v", cfg)
 
+	err := cfg.Validate()
+	if err != nil {
+		return err
```

Comment:
Might want to use errors.Wrapf(err, "%s: validation failed", cfg.Path) here just so it's crystal clear to the user that there is a config file they need to fix.

Reply:
Done.